### DISCUSSION TITLE
Why does LocalSend need my location on Windows?

### DISCUSSION THREAD

**Discussion by infinitepower18** [localsend/localsend#1376] [Discussion (English, International)]
Why does it need to access my location on Windows? It works fine without it.

![Screenshot 2024-05-25 170216](https://github.com/localsend/localsend/assets/44692189/96a737d6-df03-45ff-83ca-d48931d9de8c)

**Comment by Tienisto** [comment]
LocalSend scans your wifi network to detect other LocalSend devices. The scanning process may be interpreted as "location access".
